samples/sensor/die_temp_polling/boards/mimxrt1064_evk.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2025 SECO Mind Srl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&tempmon {
+	status = "okay";
+};
```

Comment:
I think this should go on the board file, don't see the point of having it as an overlay of the sample, though other boards are doing it too, may need a followup to clean this up

Reply:
What do you mean in the board file? 
I think it's appropriate for a sample to change the status of the module it uses.

Reply:
I mean I'd just put this straight in `zephyrproject/zephyr/boards/nxp/mimxrt1064_evk/mimxrt1064_evk.dts`, don't see the point in laving it disabled in general and enabled only for this sample

Reply:
I'm not sure about the best practices for this type of device tree node.
Also, I don't know if enabling it by default could lead to side effects on other hardware modules.
For this PR, I would prefer to stay in line with the other boards.

Reply:
yeah we don't have a solid story for that, anyway yeah fine with the current approach for this PR